Upgrade React, the Docerk base image, and add `wicked_pdf`

siefkenj opened the pull request:
Upgrade react version (and some other outdated packages), upgrade the base docker image version, and add `wicked_pdf` which allows for server-side pdf generation from html templates.

gabriellesc commented:
@jmzaleski this should not have been squashed! these commits do not exist in `master` as a result...

siefkenj committed:
Use `docker-compose.override.yml` for dev environment

	Docker will automatically pull in `docker-compose.yml`
	and `docker-compose.override.yml` when running `docker-compose
	up`. The default for `docker-compose up` should be to start
	the development environment.

	Also fixes a bug due to the an old `I18n` configuration.

siefkenj committed:
Merge remote-tracking branch 'upstream/master' into configfixes

siefkenj committed:
Upgraded React and added wicked_pdf gem

	`wicked_pdf` allows for server-side rendering of
	pdf files via html templates.

siefkenj committed:
Added wicked_pdf configuration file
